paperbackup-restore.sh

```diff
@@ -1,33 +1,38 @@
-#!/bin/bash
```

Comment:
i would argue for rewriting this script in plain POSIX instead of trying to guess where the right utilities are.